WDAC-Policy-Wizard/app/Resources/RulesDict.xml

```diff
@@ -5,146 +5,167 @@
     AllowedValue="Enabled" 
     Supported="True"
     RuleNumber="0"
+	  ValidSupplemental="False"
     ButtonMapping="button_11">
   </Rule>
   <Rule
     Name="BootMenuProtection"
     AllowedValue="Enabled"
     Supported="False"
     RuleNumber="1"
+	  ValidSupplemental="False"
     ButtonMapping="button_14">
   </Rule>
   <Rule
     Name="WHQL"
     AllowedValue="Required"
     Supported="True"
     RuleNumber="2"
-    ButtonMapping="button_8">
+    ValidSupplemental="False"
+	  ButtonMapping="button_8">
   </Rule>
   <Rule
     Name="AuditMode"
     AllowedValue="Enabled"
     Supported="True"
     RuleNumber="3"
-    ButtonMapping="button_6">
+    ValidSupplemental="True"
+	  ButtonMapping="button_6">
   </Rule>
   <Rule
     Name="FlightSigning"
     AllowedValue="Disabled"
     Supported="True"
     RuleNumber="4"
-    ButtonMapping="button_15">
+    ValidSupplemental="True"
+	  ButtonMapping="button_15">
   </Rule>
   <Rule
     Name="InheritDefaultPolicy"
     AllowedValue="Enabled"
     Supported="False"
     RuleNumber="5"
-    ButtonMapping="button_4">
+    ValidSupplemental="True"
+	  ButtonMapping="button_4">
   </Rule>
   <Rule
     Name="UnsignedSystemIntegrityPolicy"
     AllowedValue="Enabled"
     Supported="True"
     RuleNumber="6"
-    ButtonMapping="button_10">
+    ValidSupplemental="True"
+	  ButtonMapping="button_10">
   </Rule>
   <Rule
     Name="DebugPolicyAugmented"
     AllowedValue="Allowed"
     Supported="False"
     RuleNumber="7"
-    ButtonMapping="button_12">
+    ValidSupplemental="True"
+	  ButtonMapping="button_12">
   </Rule>
   <Rule
     Name="EVSigners"
     AllowedValue="Required"
     Supported="True"
     RuleNumber="8"
-    ButtonMapping="button_19">
+    ValidSupplemental="False"
+	  ButtonMapping="button_19">
   </Rule>
   <Rule
     Name="AdvancedBootOptionsMenu"
     AllowedValue="Enabled"
     Supported="True"
     RuleNumber="9"
-    ButtonMapping="button_0">
+    ValidSupplemental="False"
+	  ButtonMapping="button_0">
   </Rule>
   <Rule
     Name="BootAuditOnFailure"
     AllowedValue="Enabled"
     Supported="True"
     RuleNumber="10"
-    ButtonMapping="button_13">
+    ValidSupplemental="True"
+	  ButtonMapping="button_13">
   </Rule>
   <Rule
     Name="ScriptEnforcement"
     AllowedValue="Disabled"
     Supported="True"
     RuleNumber="11"
-    ButtonMapping="button_2">
+    ValidSupplemental="False"
```

Comment:
I assumed this **would** be configurable by a supplemental. Are you sure this is right?